app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/model/Event.kt

```diff
@@ -34,6 +35,18 @@ data class Event(@PrimaryKey
         const val defaultDuration = 7200000 // Milliseconds
     }
 
+    fun getFormattedDateTime(): String {
+        return "${getFormattedDate()}, ${getFormattedTime()}"
+    }
+
+    fun getFormattedDate(): String {
+        return DateTimeFormat.forPattern("dd MMMM yyyy").print(start)
```

Comment:
return DateTimeFormat.mediumDate().print(start)

"05.06.2018" vs "Jun 5, 2018"

Reply:
@kordianbruck I think this actually respects the default locale by default.

Reply:
Yes, it kind of does. It prints
"05 Juni 2018" vs "05 June 2018".
However, to me it seems nicer to me to have
"05.06.2018" vs. "June 05, 2018" what my suggestion does if I remember correctly